undetected_chromedriver/__init__.py

```diff
@@ -371,9 +371,9 @@ def __init__(
             options.arguments.extend(["--no-sandbox", "--test-type"])
 
         if headless or options.headless:
-            if self.patcher.version_main < 108:
+            if self.patcher.version_main is None or self.patcher.version_main < 108:
```

Comment:
If `version_main is None`, should we not rather default to `--headless=new`?

Reply:
Do you want me to change it ?